src/absil/ilwrite.fsi

```diff
@@ -15,17 +15,22 @@ type ILStrongNameSigner =
     static member OpenKeyPairFile: string -> ILStrongNameSigner
     static member OpenKeyContainer: string -> ILStrongNameSigner
 
+type EmitStreamProvider = Lazy<Choice<EmitTo, Diagnostic>>
```

Comment:
@enricosada Using Lazy in this way is not normal in the compiler.  Code that relies on observing whether lazy values have been created or not is just too subtle.  

I'd rather you somehow made the has-this-stream-been-created information more explicit somehow, rather than relying on Lazy.IsValueCreated.  

thanks